src/prefs/PlaybackPrefs.cpp

```diff
@@ -152,6 +152,7 @@ void PlaybackPrefs::PopulateOrExchange(ShuttleGui & S)
          S.TieCheckBox(XXO("Always scrub un&pinned"),
             {UnpinnedScrubbingPreferenceKey(),
              UnpinnedScrubbingPreferenceDefault()});
+         S.TieCheckBox(XXO("Send MTC (MIDI Time Code)"), {"/AudioIO/SendMTC", true});
```

Comment:
I would make a named BoolSetting object in this file, declared in a header, to avoid repetition of the string literal.  See other examples